zips/zip-0216.rst

```diff
@@ -205,6 +205,25 @@ the protocol specification [#protocol-2023.4.0]_:
   `librustzcash` implementation in all cases, which reflects the specification above.
   `zebra` always used the `librustzcash` implementation.
 
+The protocol specification was further updated in version 2025.6.0 [#protocol-2025.6.0]_,
+to tighten the type of $\mathsf{ivk}$ in Sapling to
+$\{ 1\,..\, 2^{\ell^{\mathsf{Sapling}}_{\mathsf{ivk}}}\!-1 \}$ and the type of
+$\mathsf{pk_d}$ in Sapling to $\mathsf{KA^{Sapling}.PublicPrimeOrder}$, in order to make
+the exclusion of the zero point more obvious [#zips-issue664]_. This also has the effect
+that a Sapling incoming viewing key [#protocol-saplinginviewingkeyencoding]_ or a Sapling
+IVK Encoding in a Unified Incoming Viewing Key [#zip-0316]_ that encodes the zero
+$\mathsf{ivk}$ MUST be considered invalid when imported. Another note was also added
+explicitly covering the encoding of $\mathsf{pk_d}$ in Sapling payment addresses
+[#protocol-saplingpaymentaddrencoding]_:
+
+  The restriction on $\mathsf{pk_d}$ reflects its current type
+  $\mathsf{KA^{Sapling}.PublicPrimeOrder} = \mathbb{J}^{(r)*}$. In versions of this
+  specification prior to 2025.6.0, $\mathsf{pk_d}$ had type
+  $\mathsf{KA^{Sapling}.PublicPrimeSubgroup} = \mathbb{J}^{(r)}$, i.e. including
+  $\mathcal{O}_{\mathbb{J}}$. Implementations of consumers for this encoding may need
+  to be updated to exclude $\mathcal{O}_{\mathbb{J}}$, and should be checked for
+  consistency with the current version of [ZIP-216].
```

Comment:
In a pairing, @daira and I checked `sapling-crypto` and it does prevent constructing a `PaymentAddress` with `pk_d.is_identity()`. The `DiversifiedTransmissionKey` type does allow parsing the identity, but [is documented](https://github.com/zcash/sapling-crypto/blob/6a8282be0959b410a0b622cd5eb84f8c3c134078/src/keys.rs#L475-L484) as doing so because that is the type matching the protocol spec, and it explicitly notes that rejection happens in `PaymentAddress`. We also confirmed that this rejection is done as part of note decryption.